loadtest/control/simplecontroller/controller.go

```diff
@@ -86,6 +88,16 @@ func (c *SimpleController) Run() {
 				return resp
 			},
 		},
+		{
+			run: func(user.User) control.UserActionResponse {
+				resp := control.Login(c.admin)
+				if resp.Err != nil {
+					return resp
+				}
+				c.connect()
+				return resp
+			},
+		},
```

Comment:
This is going to run for each user, so we essentially end up logging in the admin N times. I don't think we should change the controller since any user is equal from its point of view. What I would suggest is to do something even easier like making the first created user an admin in `NewControllerWrapper()`.

Reply:
That will work as well, my thinking is that each user basically can perform admin actions, so we have to log admin in N times, but you are right essentially that admin is a single user. I will update the controller to basically make the first user to be an admin

Reply:
Yes, the problem is that the controller should really be responsible for one user session only otherwise we end up making things more complex and potentially less realistic. 

I apologize for the confusion, it's totally okay to simply add a check on creation and make the first controlled user an admin.

Reply:
i think that should do it 😄 https://github.com/mattermost/mattermost-load-test-ng/pull/412/commits/b3acb8507b91a5ee66402bc4c21e23f536257f64